client/my-sites/site-settings/header-code/style.scss

```diff
@@ -0,0 +1,51 @@
+.header-code__warning {
+	display: flex;
+	margin-bottom: 1.5em;
+
+	.gridicon {
+		fill: var(--color-text-subtle);
+		margin-right: 6px;
+	}
+
+	.header-code__warning-text {
+		color: var(--color-text-subtle);
+		font-size: $font-body-extra-small;
+		margin: 0;
+	}
+}
+
+.header-code__editor .cm-editor {
+	border: 1px solid var(--studio-gray-5);
+	font-size: $font-body-small;
+	margin-bottom: 1em;
+	outline: none;
+
+	&.cm-focused {
+		border-color: var(--studio-gray-10);
+	}
+
+	* {
+		font-family: $code;
+	}
+}
+
+.header-code__editor.is-loading .cm-editor {
+	@include placeholder();
+	pointer-events: none;
+
+	&:after {
+		display: none;
+	}
+
+	.cm-line {
+		visibility: hidden;
+	}
+}
+
+.header-code__text {
+	margin-bottom: 1em;
+}
+
+.header-code__button {
```

Comment:
The less CSS we use, and the more existing components we reuse, the better.

Reply:
Am I missing a component that I should be using? 😅

Reply:
Just the button, without any alignment. Let's try to reuse as much as possible from the rest of Calypso settings, even if the design changes a little bit.

Reply:
Ah, sure!